README.md

```diff
@@ -4,7 +4,7 @@
   </a>
 </p>
 
-[![Build Status](https://travis-ci.org/meandavejustice/min-vid.svg?branch=master)](https://travis-ci.org/meandavejustice/min-vid)
+[![Build Status](https://travis-ci.org/meandavejustice/min-vid.svg?branch=master)](https://travis-ci.org/meandavejustice/min-vid) [![Available on Test Pilot](https://img.shields.io/badge/available_on-Test_Pilot-0996F8.svg)](https://testpilot.firefox.com/experiments/)
```

Comment:
Actually, mildly interesting and now slightly more related to this repo:

- https://testpilot.firefox.com/experiments works.
- https://testpilot.firefox.com/experiments/ redirects to https://testpilot.firefox.com/.